specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/examples/apps/SuccessfulGetApplicationsListRequest.json

```diff
@@ -2,8 +2,8 @@
   "parameters": {
     "Endpoint": "{Endpoint}",
     "Ocp-Apim-Subscription-Key": "00000000000000000000000000000000",
-    "skip": "0",
-    "take": "100"
+    "skip": 0,
+    "take": 100
```

Comment:
This looks like a breaking change. Has the SDK shipped GA?

Reply:
This is only updating the example. The API format itself has not changed.